### PR TITLE
Created the SPA for the math magician app

### PR DESCRIPTION
**Pull request by summyalena**
### I finished the task on SPA and followed the below guidelines.

- [ ] Add your own styles to improve the look and feel.
- [ ] In the lesson about styling, we introduced several ways to add styles in a React application:
- [ ] React basic styling (inline styles, importing CSS and pre-processed CSS files).
- [ ] CSS modules.
- [ ] Styled components.
- [ ] In this project, you should use React basic styling.
- [ ] For every page, you should create a route.
- [ ] Make sure that navigation links for all 3 routes are displayed on each page.
- [ ] Use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <NewCalculator />
+         <Headers />
+       <Home/>
       </div>
     );
   }
```
Comment (fobadara) [inline src/App.js:15]: - [ ] Good job so far but I believe you can do better with the design. You can search the internet for ideas. Try to add more colours and animations. [optional]

**Review by fobadara — APPROVED** [review]
Hi  @summyalena , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Descriptive PR

## Optional suggestions
Kindly take note of these points because they can't be included in inline comments.

- [ ] I noticed you have just one commit. Try to make more commits. With the number of changes you made I would have expected about two to three commits. Multiple commits and checkouts have saved me countless times when I want to revert changes [optional]
- [ ] Good job 👍🏽 writing a descriptive PR. But you don't have to write the body in the imperative form. Just your commits and PR title need to be in the imperative form.[optional]
- [ ] I noticed that your branch name is not descriptive enough. It should be a feature name or action for example improved_UI or aesthetics.  [optional]
- [ ] Your README can be more descriptive. You can follow this [link](https://github.com/fobadara/math-magicians-react) for some ideas. [optional]
- [ ] Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (summyalena) [reply src/App.js:15]: alright. I would do better with the design. Thank you for all other suggestions too. Would put it into considerations

Comment (fobadara) [reply src/App.js:15]: 👍🏽

**Pull request merged**